src/Middleware/RedirectToLogin.php

```diff
@@ -29,19 +30,22 @@
  */
 class RedirectToLogin implements MiddlewareInterface
 {
+    public UrlStore $UrlStore; //added this
     private Horde_Registry $registry;
     private ResponseFactoryInterface $responseFactory;
     public function __construct(Horde_Registry $registry, ResponseFactoryInterface $responseFactory)
     {
         $this->registry = $registry;
         $this->responseFactory = $responseFactory;
+        //$this->UrlStore; //added this; is it necessary?
```

Comment:
`$this->urlStore = $urlStore;` is needed.
Also needs to be added to the `__construct` parameters.
Look how the other two dependencies are added.